esm/interface/element.js

```diff
@@ -228,6 +229,9 @@ export class Element extends ParentNode {
     if (name === 'class')
       return this.className;
     const attribute = this.getAttributeNode(name);
+    if (this.ownerDocument[MIME].docType.startsWith('<?xml')) {
```

Comment:
I'm not sure what the ideal way to check if the element is part of an XML document, having it like this for now to be corrected

Reply:
Another way I see might be `this.ownerDocument[MIME].docType === Mime['application/xml'].docType` (`this.ownerDocument[MIME] === Mime['application/xml']` doesn't seem to work)